Assets/Scripts/Cheat/Main/CheatController.cs

```diff
@@ -9,7 +9,7 @@
 public class CheatController : Controller, ICheatController {
 	const string MINI_ORB_CHEAT_ID = "Increase Mini Orbs";
 
-	public int MiniOrbCheatIncreaseAmount = 100;
+	public int MiniOrbCheatIncreaseAmount = 10000;
```

Comment:
This looks good, however you also need to change this value in the inspector because it's serialized in the Unity scene. Additionally, can you update the cheats button so that it reads the correct value?